Docs v2.0 (#3)
nvnieuwk/nf-validation: docs/nextflow_schema/sample_sheet_schema_specification.md

description: Schema specification for sample sheet validation
---

> # Sample sheetschema specification

Reviewer: ```suggestion
# Sample sheet schema specification
```

Author: Good spot! thanks!

!!! tip

    Sample sheets are commonly used to define input file paths.

>     Be sure to set `"type": "string"`, `exists: true` and `"format": "file-path"`, `"schema":"path/to/samplesheet/schema.json"` for these properties,

Reviewer: ```suggestion
    Be sure to set `"type": "string"`, `exists: true`, `"format": "file-path"` and `"schema":"path/to/samplesheet/schema.json"` for these properties,
```

>     so that `fromSamplesheet` will not result in any errors.

Reviewer: ```suggestion
    so that samplesheets are correctly validated and `fromSamplesheet` does not result in any errors.
```
